docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -81,21 +89,21 @@ The Vitamin A deficiency risk factor in GBD 2017 is a **dichotomous variable** .
 Below is a list of measures and corresponding IDs:
 
 .. list-table:: Measures
-	:widths: 20 45 40
-	:header-rows: 1
-
-	* - Measure
-	  - ID
-	  - Data source
-	* - Remission
-	  - gbd_id = 2510, measure_id = 7
-	  - epi, use get_model_results function
-	* - Incidence rate
-	  - gbd_id = cid(389)
-	  - como, use get_measure
-	* - Risk factor exposure
-	  - gbd_id = reiid(96)
-	  - como, use get_measure
+  :widths: 20 45 40
+  :header-rows: 1
+
+  * - Measure
+    - ID
+    - Data source
+  * - Remission
```

Comment:
1. Incidence and remission are measures for the cause.
2. I thought we decided not to use them at all.

Reply:
Adding a todo to address this.